PWGEM/Dilepton/Core/DielectronCut.h

```diff
@@ -441,8 +469,9 @@ class DielectronCut : public TNamed
   std::function<float(float)> mMaxPhivPairMeeDep{}; // max phiv as a function of mee
   bool mSelectPC{false};                            // flag to select photon conversion used in mMaxPhivPairMeeDep
   bool mApplydEtadPhi{false};                       // flag to apply deta, dphi cut between 2 tracks
-  float mMinDeltaEta{0.f};
-  float mMinDeltaPhi{0.f};
+  float mMinDeltaEta{0.f}, mMaxDeltaEta{1e10f};
+  float mMinDeltaPhi{0.f}, mMaxDeltaPhi{1e10f};
```

Comment:
default value of mMinDeltaPhi should be -1e+10f now.

Reply:
fixed